code/modules/materials/Mat_FabRecipes.dm

```diff
@@ -75,10 +75,18 @@
 			var/obj/item/clothing/head/helmet/space/custom/helmet = new()
 			suit.set_loc(getOutputLocation(owner))
 			helmet.set_loc(getOutputLocation(owner))
-			var/obj/item/fab = getObjectByPartName("Fabric")
+			var/obj/item/fabr = getObjectByPartName("Fabric")
 			var/obj/item/vis = getObjectByPartName("Visor")
-			suit.setMaterial(fab.material)
-			helmet.setMaterial(vis.material)
+			var/obj/item/renf = getObjectByPartName("Reinforcement")
+			suit.setMaterial(fabr.material)// suit stuff
```

Comment:
Oh actually it seems you already set the base material, nice. So just change `setupSuitProp(F, R)` into `setupReinforcement(reinforcementMat)`.